news/2023/2023-01-19-gmt-apps-are-now-open.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Global Moderation Team applications are now open!
+date: 2023-01-19 12:00:00 +0000
+---
+
+Been an active community member for a while? Felt an inner urge to chase after some osu! troublemakers? Now it's your time to shine! The Global Moderation Team is seeking new members.
+
+![](/wiki/shared/news/banners/gmt-generic.jpg)
+
+It's been over 2 years since the [last GMT application cycle](https://osu.ppy.sh/home/news/2020-10-04-global-moderation-team-applications-open), and while some things changed around, one thing remains the same — we may be looking for you!
+
+The [Global Moderation Team](https://osu.ppy.sh/groups/4) is responsible for keeping the community in a good condition, with a place for every single being. The GMT is covering a handful of niches, most notable ones being in-game chat and forum moderation, let's not ignore the other parts though. The greennamed peers are also there to help moderate osu! sub communities, such as beatmapping one. The members of the team will also give you assistance with some of your technical issues. The inherent goal for us is to reach closer and closer to the living community.
```

Comment:
not entirely sure if this is what you meant by "beatmapping" as a "subcommunity"

```suggestion
The [Global Moderation Team](https://osu.ppy.sh/groups/4) is responsible for keeping the community in a good condition, keeping it a welcoming place for everyone. Our green-coloured peers of the GMT help moderate the in-game chat, forums, as well as beatmap discussions. The members of the team can also give you assistance with some of your technical issues. The inherent goal for is to reach closer and closer to the living community.
```

Reply:
i wanted to emphasize the community aspect strongly here, mapping/modding moderation are not explicitly beatmap discussions moderators, they are also responsible for giving feedback in mapping related stuffs, participating in nat discussions etc.

Reply:
so something like this?

```suggestion
The [Global Moderation Team](https://osu.ppy.sh/groups/4) is responsible for keeping the community in a good condition, keeping it a welcoming place for everyone. Our green-coloured peers of the GMT not only help moderate the in-game chat and forums, but also engage with the mapping community among others. The members of the team can also give you assistance with some of your technical issues. The inherent goal for is to reach closer and closer to the living community.
```

Reply:
oh yeah that's perfect